backend/models/listingModel.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const listingSchema = new Schema({
+  organisationName:{
+    type: String,
+    required:true
+  },
+  title: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required:true
+  },
+  location: {
```

Comment:
Hey Nadia and Ryan,
for the location property, me and Andreea had a look into this and think that the property should look like something below
`address: {
    firstLine: "Unit 2FG, Zetland House",
    City: "London",
    Postcode: "EC2A 4HJ"
  }`

This will  help when we implement search functionality for filter on i.e. postcode

So the property should be like
`address: {
    firstLine:  {
            type: String,
            required: true
        },
    city: String  {
            type: String,
            required: true
        },
    postcode: {
            type: String,
            uppercase: true,
            required: true
        }
  }`

Here is a link we found on Mongoose Schema Address: https://coursework.vschool.io/mongoose-schemas/